astroplan/core.py

```diff
@@ -25,82 +40,137 @@ class Observer(object):
     Some comments.
     """
 
-    def __init__(self, name, longitude, latitude, elevation, pressure,
-                 relative_humidity, temperature, timezone, description=None):
+    def __init__(self, name=None, location=None, latitude=None, longitude=None,
+                 elevation=None, timezone='UTC', pressure=None,
+                 relative_humidity=None, temperature=None, description=None):
         """
         Initializes an Observer object.
 
-        <longer description>
+        TODO: <longer description>
 
         Parameters
         ----------
         name : str
             A short name for the telescope, observatory or location.
 
-        longitude : str or `~astropy.units.quantity`?
-            The longitude of the observing location.
+        pressure : `~astropy.units.Quantity`
+            The ambient pressure.
 
-        latitude : str or `~astropy.units.quantity`?
-            The latitude of the observing location.
+        location : `~astropy.coordinates.EarthLocation`
+            The location (latitude, longitude, elevation) of the observatory.
+
+        longitude : str or `~astropy.units.Quantity`
```

Comment:
Maybe add `(optional)` after `~astropy.units.Quantity`, and in the first line of the description specify that this is not the preferred method for instantiating an `Observer` (e.g., use `location=`). Same below for `latitude`